docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+web:
+  build: .
+  env_file: .env
+  links:
+    - db
+  expose:
+    - "3000"
+  environment:
+    PORT: 3000
+    SERVICE_3000_CHECK_HTTP: '/health'
+    SERVICE_3000_CHECK_INTERVAL: '1s'
+db:
+  image: mysql
+  volumes_from:
+    - data-jsperf-mysql
+  env_file: .env
+  environment:
+    MYSQL_RANDOM_ROOT_PASSWORD: 'True'
+    MYSQL_PASSWORD: $MYSQL_PASSWORD
+    MYSQL_DATABASE: jsperf
+    MYSQL_USER: jsperf
+lb:
+  build: ./docker
+  dockerfile: Dockerfile-lb
+  net: 'host'
+  ports:
+    - '80:80'
+consul:
+  image: gliderlabs/consul-server:0.6
+  command: [-bootstrap, -advertise, 127.0.0.1, -client, 0.0.0.0]
+  net: 'host'
+  ports:
+   - '8500'
+   - '53:8600/udp'
+   - '8400:8400'
+registrator:
+  image: gliderlabs/registrator:v6
+  command: -internal consul://localhost:8500
```

Comment:
it looks like a lot of the container linking that docker-compose magically does went away (`localhost` instead of named host `consul` after `consul` container defined above). what happened?